frontend/src/constants/routes.js

```diff
@@ -22,9 +22,12 @@ export const MINE_HOME_PAGE = {
   route: "/dashboard/mines",
   dynamicRoute: ({ page, per_page, ...params }) =>
     `/dashboard/mines/?${queryString.stringify({ page, per_page, ...params }, { sort: false })}`,
-  mapRoute: (page, perPage, search = null) => {
-    const searchParam = search ? `&search=${search}` : "";
-    return `/dashboard/mines?page=${page}&per_page=${perPage}${searchParam}&map=true`;
+  mapRoute: (page, perPage, search = null, lat = null, long = null, zoom = null) => {
+    let searchParam = search ? `&search=${search}` : "";
+    searchParam += lat ? `&lat=${lat}` : "";
+    searchParam += long ? `&long=${long}` : "";
+    searchParam += zoom ? `&zoom=${zoom}` : "";
+    return `/dashboard/mines?${searchParam}&map=true`;
```

Comment:
Positional arguments need to be updated, as mentioned in my last comment. An use `queryString` instead of manually concatenating all of the query string params together. You can look at `dynamicRoute` for an example of how to do this

Reply:
done